point_pattern.py

```diff
@@ -33,8 +33,10 @@ def read_geojson(input_file):
     """
     # Please use the python json module (imported above)
     # to solve this one.
-    gj = None
-    return gj
+with open(input_file, 'r') as f:
+    gj = json.load(f)
+return gj
```

Comment:
bump this return so that is is indented under the `def` and resubmit.  Should be good to go then.